test/gtest/ucp/test_ucp_proto.cc

```diff
@@ -435,6 +431,8 @@ UCS_TEST_P(test_proto_perf, from_caps)
     expect_perf(2000, 2999,
                 {{UCP_PROTO_PERF_FACTOR_LOCAL_CPU, local_cpu_func}});
     expect_empty_range(3000, SIZE_MAX);
+
+    ucp_proto_perf_destroy(m_perf);
```

Comment:
Move it to `cleanup()` func for this test and remove `cleanup()` from `test_proto_perf_add_func`